File: backend/booking-service/src/main/java/com/tripify/booking_service/dto/CheckoutRequestDTO.java
package com.tripify.booking_service.dto;

import java.util.List;

// cartItemIds nullo o vuoto = checkout dell'intero carrello (comportamento
// storico); valorizzato = checkout solo di quegli articoli, lasciando gli
// altri nel carrello (vedi BookingService.checkout).
public record CheckoutRequestDTO(List<Long> cartItemIds) {}

File: frontend/app/src/main/java/com/tripify/tripify_android/data/model/CheckoutRequestDTO.kt
package com.tripify.tripify_android.data.model

// cartItemIds nullo o vuoto = checkout dell'intero carrello; valorizzato =
// checkout solo di quegli articoli, lasciando gli altri nel carrello.
data class CheckoutRequestDTO(
    val cartItemIds: List<Long>? = null
)
